recipes/r-gmcm/meta.yaml

```
@@ -0,0 +1,91 @@
+{% set version = '1.4' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-gmcm
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/GMCM_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/GMCM/GMCM_{{ version }}.tar.gz
+  sha256: dd3870981b6da33759b573a165ff4db1ea642b048494f36ba87b627a5443ba8d
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ stdlib('c') }}                # [not win]
+    - {{ stdlib('m2w64_c') }}          # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - r-rcpp >=0.10.6              # [build_platform != target_platform]
+    - r-rcpparmadillo              # [build_platform != target_platform]
+    - r-ellipse                    # [build_platform != target_platform]
+  host:
+    - libblas
+    - liblapack
+    - r-base
+    - r-rcpp >=0.10.6
+    - r-rcpparmadillo
+    - r-ellipse
+  run:
+    - r-base
+    - r-rcpp >=0.10.6
+    - r-rcpparmadillo
+    - r-ellipse
+
+test:
+  commands:
+    - $R -e "library('GMCM')"           # [not win]
+    - "\"%R%\" -e \"library('GMCM')\""  # [win]
+
+about:
+  home: https://github.com/AEBilgrau/GMCM
+  license: GPL-2.0-only
```

Shows `License: GPL (>= 2)`, so

```suggestion
  license: GPL-2.0-or-later
```